tests/test_unitary_rdws.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict
+
+import numpy as np
+
+from tests.test_unitary_base import BaseTestUnitary
+
+
+class TestRelativeDirtyWastingSeparable(BaseTestUnitary):
+    _expected_classes: Dict[str, bool] = {
+        "SCNW": False,
+        "RCNW": False,
+        "SDNW": False,
+        "RDNW": False,
+        "SCWE": True,
+        "SCWS": False,
+        "RCWS": True,
+        "SDWE": True,
+        "SDWS": False,
+        "RDWS": True,
+    }
+
+    def _take_matrix(self):
```

Comment:
This example in turn is non-wasting, because you have identity on auxiliary system